### DISCUSSION TITLE
Desenvolvimento real todos os dias ao vivo, bora lá trocar ideias

### DISCUSSION THREAD

**Discussion by wilsonneto-swe** [frontendbr/forum#2094] [Dicas]
Fala galera, tudo bem?

Comecei um projeto de uma plataforma fullstack de Blog as a Service, um projeto pessoal no qual estarei trabalhando todos os dias às 21h, e vou fazer totalmente em live ao vivo na Twitch:

Vai ter muito Node.js + TS + TDD + Clean Architecture + Express.js + Typescript + Postgres + React.js + Next.js

Link twitch: [https://www.twitch.tv/wilsonnetodev](https://www.twitch.tv/wilsonnetodev)

Bora lá programar e trocar ideia sobre carreira, desenvolvimento e muito JS!

Valeeu,
abraços!

**Comment by Kaedh** [comment]
Parei um dia pra vê sua live, achei animal!
Não sabia que tava fazendo todo dia, vou dar um follow lá e acompanhar.

**Reply by wilsonneto-swe** [reply]
Valeeu demais pelo feedback man!
